tests/test_writers.py

```diff
@@ -82,6 +83,18 @@ def test_write_sssom_json(self):
             f"{path} has the wrong number of mappings.",
         )
 
+    def test_write_sssom_jsonld(self):
+        """Test writing as JSON."""
+        path = os.path.join(test_out_dir, "test_write_sssom_json.json")
+        with open(path, "w") as file:
+            write_jsonld(self.msdf, file)
+        msdf = parse_sssom_jsonld(path)
+        self.assertEqual(
+            len(msdf.df),
+            self.mapping_count,
+            f"{path} has the wrong number of mappings.",
+        )
+
```

Comment:
Little @cthoyt in Nicos head:

> **AGAIN???**  Please add short explicit tests so I can understand what is going, in particular the difference between JSON and JSONLD serialisations.

Reply:
@matentzn: I will add tests after we have had some discussions in the nature of the JSON output.